end_to_end_test/suite_test.go

```diff
@@ -0,0 +1,110 @@
+// Copyright © 2023 Cisco Systems, Inc. and its affiliates.
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package end_to_end_test
+
+import (
+	"context"
+	"fmt"
+	"github.com/compose-spec/compose-go/cli"
+	"github.com/go-logr/logr"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
```

Comment:
just want to put this down on paper but I *hate* importing stuff with no namespacing. I get that it makes the tests read nicer, and its how that ginkgo and gomega document how to do things. but its super annoying to spend 5 minutes trying to work out where a function is coming from in the local package, only to discover that its actually through an anonymous import.

I'm not sure if this is opinion that is just a *me* problem but bleh... XD

Reply:
I agree, @sambetts-cisco. Usually, anonymous imports introduce more problems than benefits.

Reply:
thanks for the comments! yes, I agree as well 👍